src/Inventory/InventoryList.js

```diff
@@ -41,25 +86,23 @@ class InventoryList extends Component {
 			return
 		}
 
-		Meteor.call('inventories.insert', this.state.name, this.state.id, (err, newItemId) => {
+		Meteor.call('inventories.insert', this.state.name, (err, newItemId) => {
```

Comment:
How does this keep track of which list the inventory belongs to?

Reply:
Each item doesn't have a reference to what list it belongs to. The list document itself contains an array of item ids. And when we're in the list view, we just filter out any items not in the array.